src/resolvers/types/CommonWrappers.ts

```
@@ -309,7 +317,7 @@ export class ApiAccessTokenResponseWrap extends ResponseWrapBase {
 @ObjectType()
 export class ProposalsResponseWrap extends ResponseWrapBase {
   @Response()
-  @Field(() => [Proposal])
+  @Field(() => [Proposal], { nullable: true })
```

Is there any specific reason why the proposal response becomes nullable in this PR or it is a fix for something else?

---

Yes, I noticed that in case mutation fails, this field would be null.
If this is not marked as nullable you would get an GraphQL error, which says that the field can't be nullable, and this error masks the actual error and does not return rejection message. So we should always heep these fields nullable.